manifests/c/Cloudfleet/CLI/0.3.28/Cloudfleet.CLI.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Cloudfleet.CLI
+PackageVersion: 0.3.28

```

```suggestion
PackageVersion: dev
```
Please update file path to match this change.